### ISSUE TITLE
update bundle.js and bundle.css

### ISSUE THREAD

**Issue by joanise** [ReadAlongs/Studio#157]
With all the changes we're making, `readalongs/text/bundle.js` and `readalongs/text/bundle.css` and most likely no longer compatible with current output, so we should update them.

**Comment by joanise** [comment]
It would be worth having a process to update these more easily. I hacked some code in [dev.update-bundles](https://github.com/ReadAlongs/Studio/tree/dev.update-bundles) to do it in #158 but it was still pretty much semi manual.

**Comment by roedoejet** [comment]
agreed. this is done, but we'll need to do it again once we patch with https://github.com/ReadAlongs/Web-Component/pull/177

**Comment by joanise** [comment]
long done, and documented in the Studio-Web README's Build&Publish section.
